core/client/hdfs/src/main/java/alluxio/hadoop/LocalCacheFileSystem.java

```diff
@@ -54,6 +54,7 @@
     {
       add(Constants.SCHEME);
       add("ws");
+      add("hdfs");
```

Comment:
it's irrelevant to this PR

Reply:
> it's irrelevant to this PR

It is relevant in my opinion. Current implementation will cache `alluxio://` path & `ws://` path only, which doesnt apply for hive + hdfs use case